K2Bridge.Tests.UnitTests/KustoDAL/KustoResponseParserTests.cs

```diff
@@ -249,5 +269,36 @@ private static DataTable GetAggsTable()
 
             return resTable;
         }
+
+        private static DataTable GetRangeAggsTable()
+        {
+            DataTable resTable = new DataTable();
+
+            var column1 = new DataColumn("_2");
```

Comment:
As we don't use any longer _ prefix, I suggest to remove also this hack from tests.

Reply:
👍🏻